src/surfing_penguin/db_interface.py

```diff
@@ -1,4 +1,38 @@
-from surfing_penguin import api
+from surfing_penguin import api, session
+from surfing_penguin.models import User
+
+
```

Comment:
Ther User_func should be named as UserFunctions. Please take a look at https://www.python.org/dev/peps/pep-0008/